biome.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+    "vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": false },
+    "files": {
+        "ignoreUnknown": true,
+        "ignore": ["services/**/*.mjs", "node_modules", "*/**/node_modules", ".sst", ".sst*", "*/**/botocore"]
+    },
+    "formatter": {
+        "indentStyle": "space",
+        "indentWidth": 4,
+        "lineWidth": 120
+    },
+    "organizeImports": { "enabled": true },
+    "linter": {
+        "enabled": true,
+        "rules": {
+            "recommended": true,
+            "complexity": {
+                "noForEach": "off"
+            },
+            "correctness": {
+                "noUnusedImports": "error",
+                "noUnusedVariables": "error",
+                "useExhaustiveDependencies": "off"
+            },
+            "style": {
+                "useImportType": "off",
+                "useNodejsImportProtocol": "off"
+            },
+            "suspicious": {
+                "noConsoleLog": "error",
+                "noShadowRestrictedNames": "off",
+                "noArrayIndexKey": "off"
+            },
+            "a11y": {
+                "useButtonType": "off",
+                "useKeyWithClickEvents": "off"
+            }
+        },
+        "ignore": ["services/**/*.mjs", "**/node_modules", "*/**/node_modules", "**/.sst", "**/.sst*"]
```

Comment:
Can this be removed in favour of falling back to what's declared in `files` above?